src/myfunc.cc

```diff
@@ -8,7 +8,8 @@
 void
 myfunc3(void)
 {
-	spdlog::info("Malloc in lib: \n");
+	int a;
+	spdlog::info("Malloc in lib: \n", a);
 	free(malloc(1024));
```

Comment:
[llvmlibc-callee-namespace]: 'free' must resolve to a function declared within the '__llvm_libc' namespace